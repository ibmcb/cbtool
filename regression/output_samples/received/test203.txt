 status: VMC B6BFF335-FB25-5CDA-968F-77297C0D9579 was successfully registered on SimCloud "TESTCLOUD".
 status: VMC F7D1AD21-C6B3-5FCD-8F20-5FC4CEA9D4E0 was successfully registered on SimCloud "TESTCLOUD".
 status: VMC 00373061-46C7-53B1-BCB3-9CF1C2C40D2D was successfully registered on SimCloud "TESTCLOUD".
 status: VMC 517A921A-02FA-53AE-B645-AE3BA304DCC3 was successfully registered on SimCloud "TESTCLOUD".
 status: Attribute "collect_from_host" was set to "false". Skipping Host OS performance monitor daemon startup
All VMCs successfully attached to this experiment.
The following VMCs are attached to this experiment (Cloud TESTCLOUD) :
|name                    |host_count      |pool                    |cloud_hostname                  |cloud_ip         
|simzone_a               |5               |sut                     |simzone_a                       |138.65.149.179   
|simzone_b               |5               |sut                     |simzone_b                       |25.248.171.134   
|simzone_c               |5               |sut                     |simzone_c                       |182.93.204.238   
|simzone_d               |5               |lg                      |simzone_d                       |136.130.255.142  

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |38.63M                        
|Redis Version                                        |2.5.12                        
|Uptime (in seconds)                                  |13564982                      
|Total Connections Received                           |6494853                       
|Total Commands Processed                             |93756175                      
|Number of Keys                                       |260                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.0.6                         
|Storage Size                                         |16112218096.0                 
|Data Size                                            |13822917372.0                 
|Index Size                                           |550310208                     
|Average Object Size                                  |6603.17619881                 
|Collections                                          |59                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |0                             
|SVMs                                                 |0                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |8                             
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |4                             
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |0                             
|VMs ARRIVED                                          |0                             
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |0                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|SVM RESERVATIONS                                     |0                             
|SVMs ARRIVED                                         |0                             
|SVMs ARRIVING                                        |0                             
|SVMs DEPARTED                                        |0                             
|SVMs DEPARTING                                       |0                             
|SVMs FAILED                                          |0                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |32                            

 ---------------- VMC-WIDE COUNTERS ----------------
|B6BFF335-FB25-5CDA-968F-77297C0D9579 (simzone_a) VM RESERVATIONS|0                             
|517A921A-02FA-53AE-B645-AE3BA304DCC3 (simzone_d) VM RESERVATIONS|0                             
|00373061-46C7-53B1-BCB3-9CF1C2C40D2D (simzone_c) VM RESERVATIONS|0                             
|F7D1AD21-C6B3-5FCD-8F20-5FC4CEA9D4E0 (simzone_b) VM RESERVATIONS|0                             

