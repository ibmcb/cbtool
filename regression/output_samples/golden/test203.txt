 status: VMC 824A0879-5467-591B-9F13-54AD31B15F36 was successfully registered on SimCloud "TESTCLOUD".
 status: VMC 93A9DD6B-F9A6-51F2-99F7-A388188B3F81 was successfully registered on SimCloud "TESTCLOUD".
 status: VMC C57504D2-99E3-5689-B271-9028B48D3021 was successfully registered on SimCloud "TESTCLOUD".
 status: VMC 44A7B8B0-9742-5603-B0F2-888E7BFB7061 was successfully registered on SimCloud "TESTCLOUD".
 status: Attribute "collect_from_host" was set to "false". Skipping Host OS performance monitor daemon startup
All VMCs successfully attached to this experiment.
The following VMCs are attached to this experiment (Cloud TESTCLOUD) :
|name                    |host_count      |pool                    |cloud_hostname                  |cloud_ip         
|simzone_a               |5               |sut                     |simzone_a                       |34.217.142.240   
|simzone_c               |5               |sut                     |simzone_c                       |119.27.142.48    
|simzone_b               |5               |sut                     |simzone_b                       |14.22.131.129    
|simzone_d               |5               |lg                      |simzone_d                       |181.107.70.25    

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |38.63M                        
|Redis Version                                        |2.5.12                        
|Uptime (in seconds)                                  |11740225                      
|Total Connections Received                           |6492673                       
|Total Commands Processed                             |92691062                      
|Number of Keys                                       |260                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.0.6                         
|Storage Size                                         |13965987840.0                 
|Data Size                                            |12133434188.0                 
|Index Size                                           |502873056                     
|Average Object Size                                  |6346.65204926                 
|Collections                                          |61                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |0                             
|SVMs                                                 |0                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |8                             
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |4                             
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |0                             
|VMs ARRIVED                                          |0                             
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |0                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|SVM RESERVATIONS                                     |0                             
|SVMs ARRIVED                                         |0                             
|SVMs ARRIVING                                        |0                             
|SVMs DEPARTED                                        |0                             
|SVMs DEPARTING                                       |0                             
|SVMs FAILED                                          |0                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |32                            

 ---------------- VMC-WIDE COUNTERS ----------------
|93A9DD6B-F9A6-51F2-99F7-A388188B3F81 (simzone_c) VM RESERVATIONS|0                             
|44A7B8B0-9742-5603-B0F2-888E7BFB7061 (simzone_d) VM RESERVATIONS|0                             
|824A0879-5467-591B-9F13-54AD31B15F36 (simzone_a) VM RESERVATIONS|0                             
|C57504D2-99E3-5689-B271-9028B48D3021 (simzone_b) VM RESERVATIONS|0                             

